test/test_eager_transforms.py

```diff
@@ -2946,7 +2946,6 @@ def f(x: torch.Tensor) -> torch.Tensor:
             return x
         self._check_functionalize_correctness(f, torch.zeros(4, 2, device=device))
 
-
```

Comment:
My flake8 was failing on these files... Maybe different version from the one in lintrunner